Sources/PathKit.swift

```diff
@@ -563,7 +581,7 @@ extension Path {
   ///
   public func children() throws -> [Path] {
     return try Path.fileManager.contentsOfDirectory(atPath: path).map {
-      self + Path($0)
+      self + Path(direct: $0)
```

Comment:
Note this `+` in here in `children()` and also `recursiveChildren()` would previous each do the same redundant work for `self` on every addition.